### DISCUSSION TITLE
Multiple output formats

### DISCUSSION THREAD

**Discussion by eteitelbaum** [quarto-dev/quarto-cli#4093] [Feature Requests]
In R Markdown you can add some code to the YAML header in order to [knit to multiple output formats simultaneously](https://stackoverflow.com/questions/44685485/specifying-multiple-simultaneous-output-formats-in-knitr-new), e.g.: `rmarkdown::render('your.Rmd', output_format = 'all')`. 

Similarly, in Quarto, you can publish to multiple formats from the console: `quarto::quarto_render('your.qmd', output_format = 'all')`. 

I am wondering if it would be possible to make this the default behavior. In other words, the only reason I would have

```
format:
  pdf: default
  docx: default

```
in my YAML header is that I would want to publish to .pdf and .docx. So could the team make it so that both output formats are created, rather than having to specify `output_format = 'all')`?

**Comment by mcanouil** [comment]
Closing in favour of #4462 and consider <https://github.com/quarto-dev/quarto-r/issues> for the R wrapper package.
